resident/resident-service/src/test/resources/application.properties

```diff
@@ -342,6 +342,13 @@ resident.websub.authTransaction-status.topic=AUTHENTICATION_TRANSACTION_STATUS
 resident.websub.callback.authTransaction-status.relative.url=${server.servlet.context-path}/callback/authTransaction/${ida.online-verification-partner-id}
 resident.websub.callback.authTransaction-status.url=${mosipbox.public.url}${resident.websub.callback.authTransaction-status.relative.url}
 
+# WebSub Topic and callback propertis for regproc complete workflow event
+resident.websub.regproc.workflow.complete.secret=gy6pwUhKh9DE0EZn9kQDKP

```

Comment:
@kameshsr , `resident.websub.regproc.workflow.complete.secret` has to come from environment. You will need to raise DSD so that devops will update their deployment script for that.